src/main/java/frc/robot/subsystems/Shuffleboard.java

```diff
@@ -0,0 +1,109 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.cameraserver.CameraServer;
+import edu.wpi.first.wpilibj.Timer;
+
+public class Shuffleboard extends SubsystemBase {
+  private final Joystick joystick = new Joystick(0);
+  private final Timer m_timer = new Timer();
+  private int SecondsLeft = 150;
+
+  private static Shuffleboard INSTANCE = new Shuffleboard();
+
+  /**
+   * Creates a new Shuffleboard.
+   */
+  private Shuffleboard() {
+    CameraServer.getInstance().startAutomaticCapture();
+    m_timer.start();
+    SmartDashboard.putNumber("Time Left", SecondsLeft);
+  }
+
+  public static Shuffleboard getINSTANCE() {
+    return INSTANCE;
+  }
+
+  @Override
+  public void periodic() {
+
+    // This method will be called once per scheduler run
+    // shift alt F
```

Comment:
Suggest you remove this comment as it isn't relevant to your code. Maybe write it down in your notebook or somewhere if you want to remember. I'm glad you're using autoformat, I think others should use it too.

Reply:
Understood